src/polyswarmd/eth.py

```diff
@@ -16,6 +16,9 @@
 from polyswarmd import cache
 from polyswarmd.chains import chain
 from polyswarmd.response import success, failure
+from polyswarmd.websockets.messages import (Transfer, NewBounty, NewAssertion, NewVote, RevealedAssertion,
```

Comment:
I would rather do `from polyswarmd.websocket import messages` and access the classes through the namespace. I believe it makes things more clear, as users will know these are not defined in this namespace upon usage and it also helps with mocking in case we need to do it in tests, as we are not polluting this namespace.